docs/handbook/systems-administration.md

```diff
@@ -28,109 +28,207 @@ activities and many more are referred to as system administration.
+<div class="well">
+Accounts without password should not exist on the system as they could put the system at the security risk!
+</div>
+* **UID** (user ID) - unique numerical ID of the account in the system.
+The maximum value for uid is 2147483647.
+However, for compatibility reasons one should not use numbers over 65535.
+* **GID** (group ID) - unique numerical ID of the account's primary group.
+* **comment** (also referred to as gecos) - account description
+Often this filed is set to real user name.
+* **home directory** - a path where user will be after he logs into the system.
+* **login shell** - user's initial shell program.
+* **password last change time** - time when the password was lastly changed.
+* password aging information - several [shadow(4)](https://illumos.org/man/4/shadow) fields determining when password should be changed
+    * **min** - the minimum number of days required between password changes;
+    * **max** - the maximum number of days the password is valid;
+    * **warn** - the number of days before password expires that the user is warned.
+
+### Service accounts
+
+Service (system) accounts are used by applications, which are running on the system
+and are providing some services to the network such as DNS, SMTP or WWW.
+For security reasons these services are ran under non-privileged accounts.
+
+OpenIndiana comes with several service accounts such as `webservd` for web servers,
+`pkg5srv` for `pkg(5)` server or `nobody`.
+`nobody` is a system account for services needing unprivileged user.
+However, the more services are ran under this account,
+the more privileged it becomes as it gains access to service processes and files.
+
+### Roles
+
+A role is a basic unit of Role-Based access control (RBAC) or set of privileges one can assume.
+A role can be thought of as a no-login account.
+It has most of the attributes of normal user account and is identified like normal user,
+but is not allowed to log into a system directly.
+One should login using regular account and use [su(1M)](https://illumos.org/man/1M/su) to assume the role.
+
+### Superuser account
+
+Every UNIX-like system has the superuser account, named root, used for system administrative tasks.
+This account has UID 0.
+
+Using this account for everyday usage like web browsing, email reading or movie watching is not recommended
+as root account can operate without any restrictions or limits and could cause serious damage to the system.
+
+If you created user during installation, root is created as role (not a regular account).
+This role is assigned to the user created during the installation.
+This means that you can't directly log into a system using the root account.
+One has to log in as the created user and switch to root role using [su(1M)](https://illumos.org/man/1M/su).
+
+However, if the user was not created during installation, then the root is created as regular account and
+is able to log in directly.
```

Comment:
FYI, `root` also works as an account during single-user system mode, e.g. in repairs, even if defined as a role.